marlowe-tutorial/doc/marlowe-step-by-step.adoc

```diff
@@ -1,40 +1,40 @@
 
 === Close
 A contract `Close` provides for the contract to be closed (or terminated). The only action that is performs is to provide refunds to the owners of accounts that contain a positive balance. This is performed one account per step, but all accounts will be refunded in a single transaction.
 
 Before discussing other forms of contracts, we need to describe values, observations and actions.
 
 === Values, observations and actions
-*Values* include some quantities that change with time, including “the current slot number”,footnote:[The presentation here is a simplification of the concrete implementation, in which transactions are associated with a slot interval during which it is valid to add them to the blockchain.The reason for this is that in general it is difficult to predict the precise slot in which a transaction will be accepted for inclusion on the blockchain; it is therefore more robust to specify an interval in which the transaction should be accepted. The view presented here is a simplification in that effectively we consider only intervals of length one.  So, a Marlowe contract is able to access the upper and lower bounds on the current slot interval, rather than the specific current slot value. Executing a contract can, in some circumstances, lead to an “ambiguous slot interval error”, but we do not cover that any further here.] “the current balance of an account (in Lovelace)”, and any choices that have already been made; we call these _volatile_ values. Values can also be combined using addition, subtraction and negation.
+*Values* include some quantities that change with time, including “the current slot number”,footnote:[The presentation here is a simplification of the concrete implementation, in which transactions are associated with a slot interval during which it is valid to add them to the blockchain.The reason for this is that in general it is difficult to predict the precise slot in which a transaction will be accepted for inclusion on the blockchain; it is therefore more robust to specify an interval in which the transaction should be accepted. The view presented here is a simplification in that effectively we consider only intervals of length one.  So, a Marlowe contract is able to access the upper and lower bounds on the current slot interval, rather than the specific current slot value. Executing a contract can, in some circumstances, lead to an “ambiguous slot interval error”, but we do not cover that any further here.] “the current balance of some token in an account”, and any choices that have already been made; we call these _volatile_ values. Values can also be combined using addition, subtraction and negation.
```

Comment:
What is the difference here?

Reply:
I think it's just my editor removed trailing spaces. I didn't change anything here.